cmd/mesh/manifest-migrate.go

```diff
@@ -15,23 +15,131 @@
 package mesh
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 
+	"github.com/ghodss/yaml"
 	"github.com/spf13/cobra"
+
+	"istio.io/operator/pkg/apis/istio/v1alpha2"
+	"istio.io/operator/pkg/kubectlcmd"
+	"istio.io/operator/pkg/translate"
+	"istio.io/operator/pkg/util"
+	"istio.io/operator/pkg/version"
+)
+
+const (
+	defaultNamespace = "istio-system"
 )
 
-func manifestMigrateCmd(rootArgs *rootArgs) *cobra.Command {
+type manifestMigrateArgs struct {
+	// migrateDir indicates whether to migrate all values.yaml files in directory
+	migrateDir bool
+	// namespace is the namespace to get the in cluster configMap
+	namespace string
```

Comment:
Can you clarify this point?

Reply:
namespace should be specified in the helm values you're reading from (istioNamespace). For in-cluster i thought it may be better to mention the map name too but probably this is unlikely to change so i think what you have is fine.